components/NewSpendingForm.js

```diff
@@ -1,7 +1,10 @@
 import styled from "styled-components";
 import {useRouter} from "next/router";
+import {useState} from "react";
+import UserFeedback from "./UserFeedback";
 
-export default function NewSpendingForm({onAddSpending}) {
+export default function NewSpendingForm({onAddSpending, push}) {
+  const [showUserFeedback, setShowUserFeedback] = useState(false);
```

Comment:
please follow the convention to prefix callback props with `on...`, a good name would be e.g. `onPushSpending`.
You already have a callback `onAddSpending` - I don't understand why an additional callback is needed at all...

Reply:
@actyralf you're right, I don't need an additional callback in this case. I've removed it. Thanks!